### ISSUE TITLE
[Feature] GUI background

### ISSUE THREAD

**Issue by Seegras** [DaFuqs/Spectrum#148]
I'm happy the icons (at least in the version in AoF6) have changed, it doesn't look that frumpy anymore. However, there is still one thing that makes it look bad: The GUI backgrounds. Please get rid of them.

Also, it's of course an accessibility thing to, it's bad for readability.

**Comment by DaFuqs** [comment]
I very much agree GUIs do need an overhaul.

If anyone has the motivation and time to create a nice themed gui that would be awesome.

**Comment by Seegras** [comment]
In the meantime, you might want to revert to the minecraft default :-)

(I personally don't like themed GUIs, it makes for a terrible mishmash within a modpack)

**Comment by sleepy-evelyn** [comment]
> In the meantime, you might want to revert to the minecraft default :-)
> 
> (I personally don't like themed GUIs, it makes for a terrible mishmash within a modpack)

I Agree with Seagras. The basic vanilla GUI background would be better if for nothing else than accessibility. Having a smooth block colour background is much easier to see and interact with. Especially when looking for slots in a GUI.
